src/board_controller/BrainAlive/BrainAlive.cpp

```diff
@@ -0,0 +1,379 @@
+            {
+                simpleble_peripheral_disconnect (brainalive_peripheral);
+            }
+        }
+        simpleble_peripheral_release_handle (brainalive_peripheral);
+        brainalive_peripheral = NULL;
+    }
+    if (brainalive_adapter != NULL)
+    {
+        simpleble_adapter_release_handle (brainalive_adapter);
+        brainalive_adapter = NULL;
+    }
+
+    for (size_t i = 0; i < current_buf.size (); i++)
+    {
+        current_buf[i].clear ();
+    }
+    current_buf.clear ();
+    new_eeg_data.clear ();
+
+    return (int)BrainFlowExitCodes::STATUS_OK;
+}
+
+int BrainAlive_Device::config_board (std::string config, std::string &response)
+{
+    return config_board (config);
+}
+
+int BrainAlive_Device::config_board (std::string config)
+{
+    if (!initialized)
+    {
+        return (int)BrainFlowExitCodes::BOARD_NOT_CREATED_ERROR;
+    }
+    uint8_t command[5];
+    size_t len = config.size ();
+    command[0] = 0x0a;
+    command[1] = config[2]<<4;
+    command[2] = 0x00;
+    command[3] = 0x00;
+    command[4] = 0x0d;
+    if (simpleble_peripheral_write_request (brainalive_peripheral, control_characteristics.first,
+            control_characteristics.second, command, sizeof (command)) != SIMPLEBLE_SUCCESS)
+    {
+        safe_logger (spdlog::level::err, "failed to send command {} to device", config.c_str ());
+        return (int)BrainFlowExitCodes::BOARD_WRITE_ERROR;
+    }
+    return (int)BrainFlowExitCodes::STATUS_OK;
+}
+void BrainAlive_Device::adapter_1_on_scan_found (
+    simpleble_adapter_t adapter, simpleble_peripheral_t peripheral)
+{
+    char *peripheral_identified = simpleble_peripheral_identifier (peripheral);
+    char *peripheral_address = simpleble_peripheral_address (peripheral);
+    bool found = false;
+    if (!params.mac_address.empty ())
+    {
+        if (strcmp (peripheral_address, params.mac_address.c_str ()) == 0)
+        {
+            found = true;
+        }
+    }
+    else
+    {
+        if (!params.serial_number.empty ())
+        {
+            if (strcmp (peripheral_identified, params.serial_number.c_str ()) == 0)
+            {
+                found = true;
+            }
+        }
+        else
+        {
+            if (strncmp (peripheral_identified, "BrainAlive", 10) == 0)
+            {
+                found = true;
+            }
+        }
+    }
+
+    safe_logger (spdlog::level::trace, "address {}", peripheral_address);
+    simpleble_free (peripheral_address);
+    safe_logger (spdlog::level::trace, "identifier {}", peripheral_identified);
+    simpleble_free (peripheral_identified);
+
+    if (found)
+    {
+        {
+            std::lock_guard<std::mutex> lk (m);
+            brainalive_peripheral = peripheral;
+        }
+        cv.notify_one ();
+    }
+    else
+    {
+        simpleble_peripheral_release_handle (peripheral);
+    }
+}
+
+void BrainAlive_Device::Read_Data (simpleble_uuid_t service, simpleble_uuid_t characteristic,
```

Comment:
could you fix the name for this method? we don't use snake case wo capital letters for methods